manifests/init.pp

```diff
@@ -0,0 +1,166 @@
+# Sets up rkhunter to be run once per day
+#
+# @param minute_upd   Cron minute for update check
+# @param hour_upd     Cron hour for update check
+# @param monthday_upd Cron monthday for update check
+# @param month_upd    Cron month for update check
+# @param weekday_upd  Cron weekday for update check
+# @param minute       Cron minute
+# @param hour         Cron hour
+# @param monthday     Cron monthday
+# @param month        Cron month
+# @param weekday      Cron weekday
+#
+# @param package_ensure                The ensure status of packages to be managed
+#
+# @param rkhunter_conf_file            The path to the conf file
+#
+# @param rkhunter_conf_file_template   Template to use for the conf file
+#
+# @param check_for_updates             Check internet for definition updates
+#
+# @author https://github.com/simp/pupmod-simp-rkhunter/graphs/contributors
+#
+class rkhunter (
+  String[1]         $minute_upd                      = '0',
+  String[1]         $hour_upd                        = '0',
+  String[1]         $monthday_upd                    = '*',
+  String[1]         $month_upd                       = '*',
+  String[1]         $weekday_upd                     = '*',
+  String[1]         $minute                          = '10',
+  String[1]         $hour                            = '0',
+  String[1]         $monthday                        = '*',
+  String[1]         $month                           = '*',
+  String[1]         $weekday                         = '*',
+  String[1]         $package_ensure                  = 'latest',
+  String[1]  $rkhunter_conf_file                    = '/etc/rkhunter.conf',
+  String[1]  $rkhunter_conf_file_template           = 'rkhunter/rkhunter-conf.erb',
```

Comment:
ah, was more having that as a var to avoid putting hard coded stuff elsewhere should the file need to be used multiple times, than thinking about people overriding it with their own.